sdk/assertion.go

```diff
@@ -131,7 +174,7 @@ type Statement struct {
 	// Schema describes the schema of the payload. (e.g. tdf)
 	Schema string `json:"schema,omitempty" validate:"required"`
 	// Value is the payload of the assertion.
-	Value string `json:"value,omitempty"  validate:"required"`
+	Value FlexibleValue `json:"value,omitempty"  validate:"required"`
```

Comment:
Hmm this will break downstream code that creates AssertionConfig objects. I think you need to make Statement marshallable/unmarshallable and handle value there instead of creating a new type here.